Code/Engine/RendererCore/Pipeline/Implementation/RenderData.cpp

```diff
@@ -81,7 +81,7 @@ void ezMsgExtractRenderData::AddRenderData(
   auto& cached = m_ExtractedRenderData.ExpandAndGetRef();
   cached.m_pRenderData = pRenderData;
   cached.m_uiCategory = category.m_uiValue;
-  cached.m_uiComponentIndex = 0xFFFF;
+  cached.m_uiComponentIndex = 0x7FFF;
```

Comment:
Previously this did overflow as it only has 15 bits. I don't actually know the rules for shortened storage casts, did this even work correctly before?

Reply:
Should be fine, the value is overwritten later anyways.